### DISCUSSION TITLE
When I tried to create a new environment in Pycharm, I found that anaconda3/envs/ "My environment name" /bin failed to open.

### DISCUSSION THREAD

**Discussion by daiwei125** [community/community#73286] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I tried to create a new environment in Pycharm, I found that anaconda3/envs/ "My environment name" /bin failed to open. What should I do？

![微信图片_20231102190033](https://github.com/community/community/assets/131502127/ccb5f72b-00f4-4b35-b13c-a3cd251b013e)

**Comment by GigaSmurf** [comment]
The bin folder is empty, not failed to open.

Maybe try `conda install -n wcf python` in your terminal or anaconda prompt.
